archiver/main.go

```diff
@@ -533,11 +534,16 @@ func getAndSaveUsers(config LambdaConfig) error {
 		return errors.New("error getting users from KnowBe4 ..." + err.Error())
 	}
 
+	currentTime := time.Now()
```

Comment:
I typically see things like this strung together, e.g. `currentTime := time.Now().Format("2006-01-02")`

Reply:
Updated style and filename extensions. I'm not aware that QuickSight can access S3 version history, and at least currently we're using Athena rather than raw S3 as a data store, but this could potentially be useful functionality if it exists.